_data/sidebars/docs_sidebar.yml

```diff
@@ -295,8 +295,20 @@ entries:
     output: web, pdf
     folderitems:
 
-    - title: Literature guide
-      url: /topic_1_page_1.html
+    - title: Overview
+      url: /literature-guide-overview.html
+      output: web, pdf
+
+    - title: Starting points
+      url: /literature-guide-starting-points.html
+      output: web, pdf
+
+    - title: preCICE features
+      url: /literature-guide-precice-features.html
+      output: web, pdf
+    
+    - title: Adapters and coupled codes
+      url: /literature-guide-adapters-and-coupled-codes.html
```

Comment:
I am not sure if we need so many small pages for each section. Let's better move everything to a single page.